demo-app/src/main/java/org/openmined/syft/demo/StandaloneDemo.kt

```diff
@@ -20,5 +20,5 @@ fun main() {
         }
     )
 
-    syft.start()
-}
\ No newline at end of file
+    syft?.start() ?: println("Something went wrong. Couldn't start Syft")
```

Comment:
The least possibility of having the null pointer the easier it will be user to debug ;p

Reply:
I don't like moving nulls around either. That's why I was thinking on using `Option`.
Another alternative would be to use a NullObject pattern and return a Syft that does nothing.

What do you suggest using if any parameter is not correct?

Reply:
For our case atm we only need to test for Keepalive right? Which we will eventually remove by taking unsigned integers. So I would go for nullable since its already there in the PR. But let's remove the check for protocol and ask the user to choose one of the possible constants/enums